test_webpush_monthly_rotation intermittently fails

Seen on travis and locally, e.g.:

```
======================================================================
ERROR: test_webpush_monthly_rotation
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/mozilla-services/autopush/.tox/py27/lib/python2.7/site-packages/twisted/internet/defer.py", line 1185, in _inlineCallbacks
    result = g.send(result)
  File "/home/travis/build/mozilla-services/autopush/autopush/tests/test_integration.py", line 1458, in test_webpush_monthly_rotation
    eq_(len(notifs), 1)
AssertionError: 0 != 1
-------------------- >> begin captured logging << --------------------
01:20:01,856 autopush.tests.test_integration: DEBUG: Send: {"channelIDs": [], "messageType": "hello", "use_webpush": true, "uaid": ""}
01:20:01,870 autopush.tests.test_integration: DEBUG: Recv: {u'messageType': u'hello', u'status': 200, u'uaid': u'8c1fd44321b4490bb4df9c0a9bf2e810', u'use_webpush': True, u'env': u'development'}
01:20:01,873 autopush.tests.test_integration: DEBUG: Send: {"channelID": "0112cae7-8400-4056-8156-9ac28d19fab6", "key": null, "messageType": "register"}
01:20:01,890 autopush.tests.test_integration: DEBUG: Recv: {u'channelID': u'0112cae7-8400-4056-8156-9ac28d19fab6', u'messageType': u'register', u'pushEndpoint': u'http://localhost:9020/wpush/v1/gAAAAABY-AzBjgX-MrhKaoFJ65YEG3dvEpFcl1s-MEbFgnVF8YQTrkaDBiTzhIsYl_6BHGQzSMU6thil3USv59ubWl6hdNGdhR_7k9jeHOsPdKwSZpD9fOOO5RW9U3i7OfwKrT1MYGKp', u'status': 200}
01:20:01,962 autopush.tests.test_integration: DEBUG: POST body: 840e5c91834841279093058e49ac8cf5
01:20:02,017 autopush.tests.test_integration: DEBUG: POST Response (201): 
01:20:02,018 autopush.tests.test_integration: DEBUG: Response Headers: [('location', 'http://localhost:9020/m/gAAAAABY-AzBWdz84DNzG2TZID4yrPylsfUgjlDF_UFaxuVOSknGaNqwV_LZGQX4n7pSZ0VuAR8o1kIFmrl7znCQ71CbEuotFnDPz3M_IRz9Kb3vNRWyyDUSIqqHujTqGCZuK8ZgSvLvSZPhMrclGgJ4szwkLsu97_JAJsHIP5MJ47rSkrDxxAk='), ('content-length', '0'), ('ttl', '200'), ('content-type', 'text/html; charset=UTF-8'), ('date', 'Thu, 20 Apr 2017 01:20:01 GMT'), ('server', 'cyclone/1.1')]
--------------------- >> end captured logging << ---------------------
```

--- a/autopush/tests/test_integration.py
+++ b/autopush/tests/test_integration.py
@@ -1263,8 +1263,7 @@ class TestWebPush(IntegrationBase):
         # Send in a notification, verify it landed in last months notification
         # table
         data = uuid.uuid4().hex
-        with self.legacy_endpoint():
-            yield client.send_notification(data=data)
+        yield client.send_notification(data=data)
         ts, notifs = yield deferToThread(lm_message.fetch_timestamp_messages,
                                          uuid.UUID(client.uaid),
                                          " ")
@@ -1366,8 +1365,7 @@ class TestWebPush(IntegrationBase):
         # Send in a notification, verify it landed in last months notification
         # table
         data = uuid.uuid4().hex
-        with self.legacy_endpoint():
-            yield client.send_notification(data=data)
+        yield client.send_notification(data=data)
         _, notifs = yield deferToThread(lm_message.fetch_timestamp_messages,
                                         uuid.UUID(client.uaid),
                                         " ")

